backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java

```diff
@@ -26,6 +27,7 @@
 
 @Data
 @Setter
-public class GetTokensForShopResponse {
+public class GetTokensResponse {
```

Comment:
Let's have an @AllArgsConstructor and use that in the service?

Reply:
Done!